doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -462,15 +462,17 @@ The following list summarizes the most important changes inherited from the upst
 
 |no_changes_yet_note|
 
-cddl-gen
-========
+cddl-gen/zcbor
```

Comment:
Regarding my first comment, this is a way better solution to present the name change.